docker/run.sh

```diff
@@ -69,4 +70,8 @@ if [[ "$(docker images -q "$image"-"$FF_GPU_BACKEND""$cuda_version_hyphen":lates
   exit 1
 fi
 
-eval docker run -it "$gpu_arg" "--shm-size=${SHM_SIZE}" "${image}-${FF_GPU_BACKEND}${cuda_version_hyphen}:latest"
+if [[ "$detached" == "ON" ]]; then
```

Comment:
we can restore the previous code in this file if we don't run the container any longer